docs/things.postman_collection.json

```
@@ -0,0 +1,539 @@
+{
+	"info": {
+		"_postman_id": "2436794b-3a84-4c13-8b92-9f6ee5f24f8d",
+		"name": "things",
```

```suggestion
		"name": "ForgeRock IoT Collection",
```
This is a bit more consistent with our ID Cloud name: https://backstage.forgerock.com/docs/idcloud/latest/_attachments/ForgeRock_Identity_Cloud.postman_collection.json